web_custom_modifier/models/ir_ui_view.py

```diff
@@ -0,0 +1,61 @@
+# © 2019 Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import json
+from lxml import etree
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Using etree to parse untrusted XML data is known to be vulnerable to XML attacks. Replace etree with the equivalent defusedxml package.](https://app.codacy.com/app/numigi/odoo-web-addons/pullRequest?prid=4709570)